event/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
```

Comment:
add field like 

- event_type for online or in-person,
- registration_deadline
- max_capacity: for limiting the number of attendees
- tags 
- event_image 
-price (if event has fee or no fee)

Reply:
Lets add venue URL as well.